feat(ul): Add Unordered List component (#84)
zenorocha/react-email: packages/ul/readme.md

![React Email ul cover](https://react-email-assets.vercel.app/ul.png)

<div align="center"><strong>@react-email/ul</strong></div>

> <div align="center">A React ul component to help build emails.</div>

Reviewer: ```suggestion
<div align="center">An unordered list component to help build emails.</div>
```

Author: Renamed!
